core/src/main/java/org/jruby/util/io/SelectExecutor.java

```diff
@@ -209,6 +209,7 @@ IRubyObject selectInternal(ThreadContext context) throws IOException {
                 RubyIO io = TypeConverter.ioGetIO(runtime, obj);
                 RubyIO write_io = io.GetWriteIO();
                 fptr = io.getOpenFileChecked();
+                /* BUG: errorKeyList (SelectionKey) can't contain fptr.fd() (ChannelFD) */
```

Comment:
This should be excluded. I suppose it objects to a comment starting with "BUG".

Reply:
Yeah, sorry, should have cleared up during the PR comment.

There exists a bug here, and I wanted to mark it. I'm happy to remove the comment from this PR. Should I file a distinct issue to look into this?

Reply:
FIled #4867 for this.

Reply:
Thank you. Remove this from the PR and we'll discuss it separately.